scripts/setup/setup.sh

```diff
@@ -1,5 +1,34 @@
 # !/bin/bash
 
+# Color Support
+if [ -t 1 ] && [ -n "$(tput colors)" ]; then
+    RED="$(tput setaf 1)"
+    GREEN="$(tput setaf 2)"
+    YELLOW="$(tput setaf 3)"
+    BLUE="$(tput setaf 4)"
+    MAGENTA="$(tput setaf 5)"
+    CYAN="$(tput setaf 6)"
+    WHITE="$(tput setaf 7)"
+    BOLD="$(tput bold)"
+    RESET="$(tput sgr0)"
+else
+    # stdout does not support colors
+    RED=""
+    GREEN=""
+    YELLOW=""
+    BLUE=""
+    MAGENTA=""
+    CYAN=""
+    WHITE=""
+    BOLD=""
+    RESET=""
+fi
+
+# Status Messages
+OK_MSG="${GREEN}[OK]     ${RESET}"
```

Comment:
For consistency, lets change the name of these variables to:
```diff
- OK_MSG="${GREEN}[OK] ${RESET}"
- WARNING_MSG="${YELLOW}[WARNING]${RESET}"
- FAIL_MSG="${RED}[FAIL]   ${RESET}"
+ OK="${GREEN}[OK] ${RESET}"
+ WARNING="${YELLOW}[WARNING]${RESET}"
+ FAIL="${RED}[FAIL]   ${RESET}"
```
Fix all references